install/running_install.adoc

```diff
@@ -454,11 +460,23 @@ playbook is run:
 |Prometheus Install
 |*_{pb-prefix}playbooks/openshift-prometheus/config.yml_*
 
+|Availability Monitoring Install
```

Comment:
In the source bug, *_{pb-prefix}playbooks/openshift-monitoring/config.yml_* is listed again before this entry. If the first instance is correct and you need to run this playbook only once, this LGTM. (It matches the approved list in the bug.)

@vikram-redhat, I think that after this point is cleared up we should merge the change - it matches the list that QE accepted in the source bug.